### ISSUE TITLE
[R66] ClassCastException thrown by EXECOUTPUT tasks the the command failed

### ISSUE THREAD

**Issue by bcarlin** [waarp/Waarp-All#59]
Here is the full stacktrace:

```
30/07/2020/12:47:31.051 DEBUG [org.waarp.openr66.protocol.localhandler.LocalServerHandler] [Handler-1] org.waarp.openr66.protocol.localhandler.LocalServerHandler.exceptionCaught(LocalServerHandler.java:370) : [server1] Exception and isFinished: false
java.lang.ClassCastException: java.lang.Thread cannot be cast to org.waarp.openr66.context.task.AllLineReader
	at org.waarp.openr66.context.task.ExecOutputTask.finalizeFromError(ExecOutputTask.java:204)
	at org.waarp.openr66.context.task.AbstractExecTask$ExecuteCommand.invoke(AbstractExecTask.java:552)
	at org.waarp.openr66.context.task.ExecOutputTask.run(ExecOutputTask.java:127)
	at org.waarp.openr66.database.data.DbTaskRunner.runNextTask(DbTaskRunner.java:2718)
	at org.waarp.openr66.database.data.DbTaskRunner.runNext(DbTaskRunner.java:2796)
	at org.waarp.openr66.database.data.DbTaskRunner.run(DbTaskRunner.java:2844)
	at org.waarp.openr66.database.data.DbTaskRunner.finalizeTransfer(DbTaskRunner.java:3035)
	at org.waarp.openr66.context.R66Session.setFinalizeTransfer(R66Session.java:953)
	at org.waarp.openr66.protocol.localhandler.TransferActions.endTransferR(TransferActions.java:1168)
	at org.waarp.openr66.protocol.localhandler.TransferActions.fromEndTransferSToTransferR(TransferActions.java:1048)
	at org.waarp.openr66.protocol.localhandler.TransferActions.endTransfer(TransferActions.java:1033)
	at org.waarp.openr66.protocol.localhandler.LocalServerHandler.channelRead1(LocalServerHandler.java:265)
	at org.waarp.openr66.protocol.localhandler.LocalServerHandler.channelRead0(LocalServerHandler.java:107)
	at org.waarp.openr66.protocol.networkhandler.NetworkServerHandler.channelRead0(NetworkServerHandler.java:392)
	at org.waarp.openr66.protocol.networkhandler.NetworkServerHandler.channelRead0(NetworkServerHandler.java:63)
	at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:99)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.access$600(AbstractChannelHandlerContext.java:61)
	at io.netty.channel.AbstractChannelHandlerContext$7.run(AbstractChannelHandlerContext.java:370)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:164)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:472)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:500)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.lang.Thread.run(Thread.java:748)
```

Steps to reproduce :

create a rule that starts the script `fail.sh`, which always ends in error :

Content of the rule:

```xml
<rule>
  <idrule>error</idrule>
  <comment>The default rule send rule (extended version)</comment>
  <mode>1</mode>
  <rpretasks>
    <tasks>
    </tasks>
  </rpretasks>
  <rposttasks>
    <tasks>
      <task>
        <type>EXECOUTPUT</type>
        <path>./bin/fail.sh</path>
        <delay>60000</delay>
      </task>
    </tasks>
  </rposttasks>
  <rerrortasks>
    <tasks>
    </tasks>
  </rerrortasks>
  <spretasks>
    <tasks>
    </tasks>
  </spretasks>
  <sposttasks>
    <tasks>
    </tasks>
  </sposttasks>
  <serrortasks>
    <tasks>
    </tasks>
  </serrortasks>
</rule>
```

content of the script:

```sh
#!/usr/bin/env bash

echo will fail
exit 34
        
```

**Comment by fredericBregier** [comment]
Fixed in #66

**Comment by fredericBregier** [comment]
Fixed in #66
